# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/notification/moz.build
# Commit: bb63792c3333
# Full Hash: bb63792c3333a7911bdfad9e9e9ec58bb48bad93
# Author: Kagami Sascha Rosylight <saschanaz@outlook.com>
# Date: 2024-11-08 04:19:40
# Regressor Bug: 1928702
# File Overlap Count: 1
# Description:
#   Bug 1928702 - Part 2: Create PNotification via PBackground r=asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D227687
# ==============================================================================

diff -r 93937caf6d13 -r bb63792c3333 dom/notification/moz.build
--- a/dom/notification/moz.build	Thu Nov 07 17:32:03 2024 +0000
+++ b/dom/notification/moz.build	Thu Nov 07 17:32:03 2024 +0000
@@ -21,6 +21,7 @@
 ]
 
 EXPORTS.mozilla.dom.notification += [
+    "IPCUtils.h",
     "NotificationChild.h",
     "NotificationParent.h",
 ]